scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh

```diff
@@ -37,7 +37,7 @@ usage()
   echo "Usage:
   dockcross-manylinux-download-cache-and-build-module-wheels
     [ -h | --help ]           show usage
-    [ -c | --cmake_options ]  space-delimited string containing CMake options to forward to the module (e.g. \"-DBUILD_TESTING=OFF\")
+    [ -c | --cmake_options ]  space-delimited string containing CMake options to forward to the module (e.g. \"--config-setting=cmake.define.CMAKE_BUILD_TYPE:STRING=Debug\")
```

Comment:
I think this is only valid with `pyproject.toml` whereas passing options with `-D` currently works with both `setup.py` and `pyproject.toml` since 1243f8df202eed65d6e5c06602373a0c1ed4efa5

Reply:
Nice, I will drop that patch in that case.